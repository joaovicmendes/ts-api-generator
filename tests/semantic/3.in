// Deve reportar identificador jรก declarado
model [
    Student {
        name: string
        email: string
        ra: number
        ra: string
    }
    ExampleClass {
        student: Student
        grade: number
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:ra'
        GET '/students/:ra'
        POST '/students'
        PUT '/students/:ra'
        DELETE '/students/:ra'
    }
    Student {
        GET 'class/:grade'
    }
]
