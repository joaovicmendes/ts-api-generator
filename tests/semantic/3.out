Linha 7: identificador ra já declarado anteriormente
Linha 18: rota GET '/students/:ra' declarado anteriormente
Linha 23: identificador Student já declarado anteriormente
Compilação interrompida.
